.github/workflows/trigger_build_push.yml

```diff
@@ -10,24 +10,33 @@ on:
       shardeum_branch:
         description: 'Shardeum repository branch'
         required: true
-        default: 'dev'
+        default: 'mainnet-launch'
       relayer_collector_branch:
         description: 'Relayer collector repository branch'
         required: true
-        default: 'dev'
+        default: 'mainnet-launch'
       json_rpc_server_branch:
         description: 'JSON RPC server repository branch'
         required: true
-        default: 'dev'
+        default: 'mainnet-launch'
       image_tag:
-        description: 'Docker image tag'
+        description: 'Docker image tag. Should be prefixed with the network name.'
         required: true
-        default: 'dev'
+        default: 'testnet-1'
       push_latest:
         description: 'Push latest tag'
         type: boolean
         required: true
         default: false
+      network:
+        type: choice
+        description: 'Network. Will use the config for the network in scripts/config if it exists. Custom if you are doing a custom build and will use environment variables passed to Docker run.'
+        required: true
+        options:
+          - mainnet
+          - stagenet
+          - testnet
+          - custom
```

Comment:
how does this work? will i get an input box to add my network manually?

Reply:
no, if you put `custom` you will need to pass the required variables to `docker run`, theres an example in the README.